addons/settings/fnc_parse.sqf

```diff
@@ -28,6 +28,30 @@ Author:
 #define ASCII_ARRAY_OPEN (toArray "[" select 0)
 #define ASCII_ARRAY_CLOSE (toArray "]" select 0)
 
+private _fnc_parseArray = {
```

Comment:
It's a stringified array. `"[<...>]"`. `parseNumber` for example also takes \<STRING\> and not \<NUMBER\>

```
parseNumber "1"
-> 1
```